tests/pip_hub_repository/render_pkg_aliases/render_pkg_aliases_test.bzl

```diff
@@ -136,24 +135,25 @@ alias(
     name = "dist_info",
     actual = select(
         {
-            "@@rules_python//python/config_settings:is_python_3.2": "@pypi_32_bar_baz//:dist_info",
+            "@@//python/config_settings:is_python_3.2": "@pypi_32_bar_baz//:dist_info",
```

Comment:
This `@rules_python//...` -> `@@//...` part of the diff is just incidental, right? Because the repo the tests are running in are the same as the target they're referring to. I'm just double checking that they'd be the proper "@rules_python//..." strings when run from a user repo.

Reply:
correct, this is incidental. Because we are doing `str(Label(...))` it should work (as is tested by the `bzlmod` example.